src/app.md

```diff
@@ -193,6 +194,63 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-tar-strip-prefix]: #ex-tar-strip-prefix
+<a name="ex-tar-strip-prefix"></a>
+## Decompress a tarball while removing a prefix from the paths
+
+[![flate2-badge]][flate2] [![tar-badge]][tar] [![cat-compression-badge]][cat-compression]
+
+Strip a path prefix from the entries of a tarball before unpacking them.
+
+In this case we have a tarball, `archive.tar.gz`, whose contents are inside the
+sub-directory `bundle/logs`. First we create a [`tar::Archive`], similar to the
```

Comment:
No need to describe exact same steps which were covered by another less advanced example (nor refer to it).
I would remove the first two sentences of this paragraph and mention the `bundle/logs` prefix in later sentences